openproblems/tasks/_cell_cell_communication/_common/methods/baseline.py

```diff
@@ -73,7 +73,7 @@ def _true_events(adata, test=False):
 
 
 @_true_method(
-    method_name="True Events SUM",
+    method_name="True Events (max)",
```

Comment:
If `any(adata.uns["ccc_target"][merge_keys].duplicated()) is False`, then do we need aggregation for true events? THey should already be unique